community/kernel/src/main/java/org/neo4j/kernel/impl/transaction/command/Command.java

```diff
@@ -313,30 +247,45 @@ public long getRelId()
 
     public static abstract class TokenCommand<RECORD extends TokenRecord> extends Command
```

Comment:
Is this class really needed? Can `PropertyKeyTokenCommand`, `RelationshipTypeTokenCommand` and `LabelTokenCommand` simply extend `BaseCommand`?